packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -231,17 +231,13 @@ class MiniCart extends Component {
     }
 
     render() {
-        if (this.props.loading) {
```

Comment:
This wasn't actually doing anything. Notice there was no `loading` prop in anything that created `<MiniCart />` nor was there anything in `mapStateToProps`.

Reply:
Good catch. There are a few dead branches sitting around, and it's nice to identify them one by one.